mypy/checkmember.py

```diff
@@ -1139,8 +1139,8 @@ def analyze_enum_class_attribute_access(
     # Skip these since Enum will remove it
     if name in ENUM_REMOVED_PROPS:
         return report_missing_attribute(mx.original_type, itype, name, mx)
-    # For other names surrendered by underscores, we don't make them Enum members
-    if name.startswith("__") and name.endswith("__") and name.replace("_", "") != "":
+    # Dunders and private names are not Enum members
```

Comment:
nit: I would call any name that starts with a single underscore a "private" (by convention) name, and any name that starts with two underscores a "mangled" name

```suggestion
    # Dunders and mangled names are not Enum members
```

(There's several other places in this PR where you also refer to them as "private" names; I'd change those too, personally)

Reply:
I used the same terminology used by the enum docs: https://docs.python.org/3/howto/enum.html#private-names

Reply:
Fair enough... though I sorta think the docs there are using the wrong terminology... but hey :P